test/Dialect/FIRRTL/parse-basic.fir

```diff
@@ -1912,3 +1912,15 @@ circuit PublicModules:
   module Bar:
   ; CHECK: firrtl.module @PublicModules
   public module PublicModules:
+
+;// -----
+
+FIRRTL version 4.0.0
+; CHECK-LABLE: firrtl.circuit "LayerEnabledModule"
+circuit LayerEnabledModule:
+  layer A, bind:
+  layer B, bind:
+    layer C, bind:
+  ; CHECK: firrtl.module @LayerEnabledModule
+  ; CHECK-SAME: layers = [@A, @B::@C]
+  module LayerEnabledModule enablelayer A enablelayer B.C:
```

Comment:
superficial suggestion: could/should these enablelayer directives be each on their own lines?

Reply:
Are you suggesting allowing the line to break (that works now!) or are you saying to make these port-like where they go on their own lines?

Reply:
My thought is to make them more like port decls. I'm fine with how it is now, too.